dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -741,11 +762,18 @@ private void injectLibertyServers() {
             Class<?> testClass = getTestClass().getJavaClass();
             try {
                 LibertyServer serv = LibertyServerFactory.getLibertyServer(serverName, testClass);
+                ServerConfiguration config = serv.getServerConfiguration();
+
+                //if the TestServlet annotation is in use on the LibertyServer, check that the componenttest feature is configured on the server
+                TestServlet testServletAnno = serverField.getAnnotation(TestServlet.class);
```

Comment:
I think you also need to check for `@TestServlets` here.

Currently `@TestServlet` is not a repeatable annotation (though it probably could be now that we require Java 8+ everywhere) so you need to check for the two annotations separately.